uberpoet/filegen.py

```diff
@@ -43,19 +43,28 @@
 }}"""
 
 
+class Language(object):
+    swift = 'Swift'
```

Comment:
In python, constants are written in all caps snake case: https://www.python.org/dev/peps/pep-0008/#constants

Reply:
I know i dont do it everywhere, but that was before i learned this was the standard :D

Reply:
haha thanks! will change.